### ISSUE TITLE
Use the commit template in the default body template

### ISSUE THREAD

**Issue by mlazowik** [topfreegames/fluxcloud#25]
Would be nice if setting the `COMMIT_TEMPLATE` env was enough to get both the title and body links working. Right now only the title link works.

**Comment by fdpeiter** [comment]
Hello there @mlazowik! Could you provide an example? Not sure if I've understood the issue

**Comment by mlazowik** [comment]
Sure, what I mean is that [the default body template](https://github.com/topfreegames/fluxcloud/blob/master/pkg/formatters/default.go#L26)  hardcodes `$.VCSLink "/commit/" .Revision`, so if I use custom links I have to override both the commit and body templates. If the default body template was reusing `commitTemplate := config.Optional(commitTemplateKey, commitTemplate)` inside it would be enough to change `COMMIT_TEMPLATE` to get links working in both the title and body.

**Comment by fdpeiter** [comment]
You basically want to insert the commitTemplate inside the bodyTemplate, right?
Like here: 
https://github.com/topfreegames/fluxcloud/blob/master/pkg/formatters/default.go#L30?

**Comment by mlazowik** [comment]
Yep, nothing super important, just a small quality of life improvement :)

**Comment by fdpeiter** [comment]
@mlazowik I was double checking if it was possible using nested templates, but couldn't find any elegant/easy way of including a nested template over an iteration of all commits: https://github.com/topfreegames/fluxcloud/blob/master/pkg/formatters/default.go#L30

Seems to be possible if we refactor the whole template "format/logic"
